Dfe.Academies.External.Web/Pages/School/LoanDetails.cshtml

```diff
@@ -0,0 +1,70 @@
+@using Dfe.Academies.External.Web.Extensions
+@using Dfe.Academies.External.Web.TagHelpers
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model Dfe.Academies.External.Web.Pages.School.LoanDetails
+@{
+	ViewData["Title"] = "Apply to become an academy - About the conversion";
+}
+@section BeforeMain
+{
+	<a asp-page="../ApplicationOverview" asp-route-appId="@Model.ApplicationId" class="govuk-back-link">Back to application overview</a>
+}
+
+<div class="govuk-grid-column-two-thirds">
+	<h1 class="govuk-heading-l">
+		<span class="govuk-caption-l">Finances (Step 5 of 6)</span>
+		Loan Details
+	</h1>
+    <form method="post" name="frmLoanDetails" novalidate="">
+        @Html.AntiForgeryToken()
+        <input type="hidden" asp-for="Urn" />
+        <input type="hidden" asp-for="ApplicationId" />
+        <input type="hidden" asp-for="Id"/>
+        <partial name="_HiddenFields" />
+        
+        <div class="@(!ViewData.ModelState.IsValid ? "govuk-form-group--error" : "")">
+	        <fieldset class="govuk-fieldset">
+		        <div class="@(!ViewData.ModelState.IsValid ? "govuk-visually-hidden" : "")">
+			        <partial name="_ValidationSummary" model="Model.ValidationErrorMessagesViewModel"/>
+		        </div>
+		        
+		        <div class="govuk-form-group">
+		            <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+			        Total amount
+			        </legend>
+			        <div class="govuk-form-group">
+					    <input type="number" asp-for="TotalAmount" id="TotalAmount" class="govuk-input govuk-input--width-20"/>
+				    </div>
+			        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+				        Purpose of loan
+					</legend>
+			        <div class="govuk-form-group">
+				        <input type="text" asp-for="Purpose" id="Purpose" class="govuk-input govuk-input--width-20"/>
+			        </div>
+			        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+				        Loan provider
+					</legend>
+			        <div class="govuk-form-group">
+				        <input type="text" asp-for="Provider" id="Purpose" class="govuk-input govuk-input--width-20"/>
+			        </div>
+		            <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+			        Interest rate
+			        </legend>
+			        <div class="govuk-form-group">
+					    <input type="decimal" asp-for="InterestRate" id="TotalAmount" class="govuk-input govuk-input--width-20"/>
+				    </div>
+			        <label for="capitalCarryForwardStatus-deficit" class="govuk-label">
+				        Schedule of Repayment
+				        </label>
+			        <textarea asp-for="RepaymentSchedule" id="RepaymentSchedule" class="govuk-textarea" cols="40" rows="5" maxlength="400"></textarea>
+		        </div>                
+		        
+	        </fieldset>
+		</div>
+		<br />
+        
+		<input type="submit" value="Save and return to overview" class="govuk-button govuk-!-margin-top-2 govuk-!-margin-bottom-8"/>
```

Comment:
before dan tests this, is this button text correct? Should it be 'continue' or 'save and continue'?

Reply:
The ticket shows 'save and continue' on the picture so I used that. Seems like the AC and the example are different